ajax/libs/cssobj/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "cssobj", 
+  "description": "Runtime CSS manager, Turn CSS into dynamic JS module, Stylesheet CRUD (Create, Read, Update, Delete) in CSSOM, Solve common problems of CSS-in-JS, name space (local) class names, media query hook for IE8, styled components",
+  "keywords": [
+    "cssom",
+    "prefixer",
+    "css generate",
+    "diff css",
+    "media query",
+    "css modules"
+  ],
+  "author": {
+    "author": "James Yang <jamesyang999@gmail.com>",
+    "web": "https://cssobj.github.io/cssobj-demo/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cssobj/cssobj.git"
+  },
+  "filename": "cssobj.js",
+  "autoupdate": {
+    "source": "git",  
+    "target": "git://github.com/cssobj/cssobj.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [	
+            "cssobj.amd.js",
+            "cssobj.cjs.js",
+            "cssobj.es.js",
+            "cssobj.iife.js",
+            "cssobj.min.js",
+            "cssobj.umd.js"
+        ]
+      }
+    ]
+  },
```

Comment:
Because there are more release versions from npm(32) than GitHub(23), so we choice npm auto-update is better. This can let people choose more versions.
Let's modify from A to B:
- A
```js
  "autoupdate": {
    "source": "git",  
    "target": "git://github.com/cssobj/cssobj.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [	
            "cssobj.amd.js",
            "cssobj.cjs.js",
            "cssobj.es.js",
            "cssobj.iife.js",
            "cssobj.min.js",
            "cssobj.umd.js"
        ]
      }
    ]
  },
```
- B
```js
  "npmName": "cssobj",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/cssobj.!(amd|cjs|es|iife)*"
      ]
    }
  ],
```

Memo: `files` is a pattern matcher allowing selection of multiple files, see [node-glob](https://github.com/isaacs/node-glob#glob-primer)